LogMonitor/LogMonitorTests/EtwMonitorTests.cpp

```diff
@@ -82,52 +82,68 @@ namespace LogMonitorTests
 
                 //
                 // The "Microsoft-Windows-User-Diagnostic" provider produces events
-                // like this:
+                // like this (without the space formatting / one-line):
                 //
-                //  <Source>EtwEvent</Source><Time>2019-11-04T18:04:33.000Z</Time>
-                //  <Provider idGuid="{305FC87B-002A-5E26-D297-60223012CA9C}"/>
-                //  <DecodingSource>DecodingSourceXMLFile</DecodingSource>
-                //  <Execution ProcessID="3968" ThreadID="2504" /><Level>Warning</Level>
-                //  <Keyword>0x0</Keyword><EventID Qualifiers="1">1</EventID>
-                //  <EventData><ErrorCode>0x3F0</ErrorCode></EventData>
+                /*

```

Comment:
do we have any tests to validate we output valid json each time? I'd like to be sure any subsequent changes to the code do not break the json structure

Reply:
Yes, for ETW and EventLog, we have tests for that.

Reply:
e.g.
![image](https://user-images.githubusercontent.com/261265/213113533-87d6ea9d-a4d8-4eda-9357-9361c8f73245.png)

Though we don't have for FileLog and Process, I have a TODO for that.